src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/project/markdown/MarkdownProjectReader.kt

```diff
@@ -87,9 +88,9 @@ class MarkdownProjectReader() : IProjectReader {
                     titleKey = "$id",
                     resourceContainer = null)
 
-    private fun content(sort: Int, label: String, id: Int, text: String): Content? =
+    private fun content(sort: Int, label: String, id: Int, text: String, type: ContentType): Content? =
```

Comment:
This is probably stating the obvious, but I suppose we could stop using the label field to distinguish titles from bodies now. I don't remember what else the label field is used for